### PR TITLE
Add ember-accessibility

### PR DESCRIPTION
**Pull request by astronomersiva**

### CODE REVIEW

**Comment by MelSumner** [comment]
Hi! Can you tell me how this differs from ember-a11y-testing?

**Comment by jenweber** [comment]
@MelSumner There is a section about it here: <https://github.com/coyote-labs/ember-accessibility#how-is-this-different-from-ember-a11y-testing>

**Comment by astronomersiva** [comment]
Hello @MelSumner,

Like given in the link provided by @jenweber, this addon is mostly geared towards helping with development rather than the testing phase. They both use Axe under the hood but ember-accessibility provides directly addressable feedback. We have found it useful in our large team as people just have to click on the accessibility violation indicator and the addon tells how to fix that issue.

Here's an example.

Screen Shot 2019-09-01 at 21 16 22

Also, both addons have test helpers in case that's relevant.

**Comment by MelSumner** [comment]
@astronomersiva ahh okay. I'd like to check the addon out first and make sure that it's accessible. Also, could you help me understand why a separate addon was created instead of the typical community approach of contributing to an addon that already existed to do the same thing? Thanks!

**Comment by jenweber** [comment]
Just to share an update, this PR is remaining open because I think there are some checks that need to happen to make sure the addon itself is accessible. Hover states are tricky. With Octane coming down the line, this is on the backburner for the next few weeks, but we'll come back to it. Thanks for your patience!

**Comment by astronomersiva** [comment]
Appreciate the update @jenweber. I'll also take a look at the addon over the weekend and see if we've missed something.

To answer @MelSumner's  question(apologies for missing it), what this addon does would have required a pretty significant rewrite of the ember-a11y-testing addon.To be frank it was quite simpler to just write something from the ground up. While the addon itself might lack accessibility in a few cases (we intend to evaluate and fix it over the next few days), we have used it on our apps in a team of a about forty developers and what we all loved was how simple the fixes were thanks to the context provided.

File: guides/release/accessibility/application-considerations.md [hunk 1]
```diff
@@ -87,5 +87,6 @@ Here are some examples of accessibility-focused addons created by many people th
 - [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
 - [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
 - [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
+- [ember-accessibility](https://github.com/coyote-labs/ember-accessibility) - An Ember.js addon to find and address accessibility violations during development.
```
Comment (MelSumner) [inline guides/release/accessibility/application-considerations.md]: please update this description to indicate that it is the same as ember-a11y-testing but with an in-browser interactive interface. Thank you!

Comment (astronomersiva) [reply guides/release/accessibility/application-considerations.md]: I have changed the wording, thanks for your time.

**Comment by MelSumner** [comment]
After looking at this more, we don't think this is ready to include for official recommendation at this time. Please DM me on Discord, and I will work with you to get it ready for inclusion. In the meantime, I am going to close this PR. Thank you!

**Pull request closed without merging**